ui/docs/reference/platform/object/aws-backend.mdx

```diff
@@ -65,4 +64,4 @@ The provided AWS credentials should have the following policy (with `<BUCKET_NAM
 
 The use of `signGetUrls` is described above.
 If this configuration value is `false`, then the bucket should be configured to allow public read access.
-AWS has many layers of "yes, I'm really quite sure" that must be clicked through to accopmlish this.
+AWS has many layers of "yes, I'm really quite sure" that must be clicked through to accomplish this.
```

Comment:
occasionally I spend a few minutes thinking about a spellcheck lint on our comments... one day

Reply:
That would kill me :(